clients/go/apiban/apiban.go

```diff
@@ -275,7 +276,7 @@ func ProcBannedResponse(entry *Entry, id string, blset ipset.IPSet) {
 				continue
 			}
 		}
-		err := blset.Add(ipStr, GetConfig().blTtl)
+		err := blset.Add(ipStr, ttl)
```

Comment:
Extra check if ok:
use ttl: either config or from answer metadata (and not always the config version)